backend/ToDosBackend/src/main/java/com/encora/ToDosBackend/controller/ToDoController.java

```diff
@@ -17,12 +18,16 @@ public class ToDoController {
     ToDoService myService;
 
     @GetMapping("/todos")
-    public ResponseEntity<List<ToDo>> getTodos(){
-        return new ResponseEntity<>(myService.getTodos(), HttpStatus.OK);
-        ///Include pagination. Pages should be of 10 elements.
-        /// Filter by done/undone
-        /// Filter by the name or part of the name
-        /// Filter by priority ///
+    public ResponseEntity<List<ToDo>> getTodos(
+            @RequestParam(required = false) String nameFilter,
+            @RequestParam(required = false) Integer priorityFilter,
+            @RequestParam(required = false) Boolean filterDone,
+            @RequestParam(required = false) Integer pagination,
+            @RequestParam(required = false) Integer orderPriority
+            //@RequestParam(required = false) String orderDate
```

Comment:
please be sure that the final version when all requirements are met don't have comments